homeassistant/components/smartthings/__init__.py

```diff
@@ -156,6 +159,20 @@
     return True
 
 
+async def async_get_entry_scenes(entry: ConfigEntry, api):
+    """Get the scenes within an integration."""
+    try:
+        return await api.scenes(location_id=entry.data[CONF_LOCATION_ID])
+    except ClientResponseError as ex:
+        if ex.status == 403:
+            _LOGGER.exception("Unable to load scenes for config entry '%s' "
```

Comment:
Any reason you don't want to act on an invalid token? Should we notify the user via a persistent_notification ? Should we trigger a new config flow to request the new scope?

Reply:
It’s a valid token, just missing an OAuth scope.  I didn’t think it needed to be an error for the whole integration or something we wanted to pester the user about.  This could be a way to “turn off scenes” in the integration.

Reply:
In the future I’ll mske the device scope optional too, so someone could only care about scenes.